code/game/jobs/job/support.dm

```diff
@@ -380,8 +381,7 @@
 		H.mind.AddSpell(new /obj/effect/proc_holder/spell/aoe/conjure/build/mime_wall(null))
 		H.mind.AddSpell(new /obj/effect/proc_holder/spell/mime/speak(null))
 		H.mind.miming = 1
-
-
+	qdel(H.GetComponent(/datum/component/footstep))
```

Comment:
I think this means that cloned mimes will still have the footstep component, won't they? Maybe just add a user.mind.miming check to /mob/living/carbon/human/Initialize

Reply:
check implemented